### ISSUE TITLE
Tailwind CSS files are not included in release

### ISSUE THREAD

**Issue by gapmiss** [nicholas-wilcox/unofficial-tailwindcss-obsidian-plugin#2]
Greetings~

Thank you for your contribution to Obsidian development.

After installing this plugin, one has to add 2 `css` files to the plugin's directory for the plugin to start working.

https://raw.githubusercontent.com/tailwindlabs/tailwindcss/master/tailwind.css
https://raw.githubusercontent.com/tailwindlabs/tailwindcss/master/src/css/preflight.css

The plugin's directory structure should be similar to:

```
└── VAULT
    └── .obsidian
        └── plugins
            └── unofficial-tailwindcss-obsidian-plugin
                ├── data.json
                ├── main.js
                ├── manifest.json
                ├── preflight.css
                └── tailwind.css
```

**Comment by nicholas-wilcox** [comment]
Ah, good catch. And thanks for reaching out before I even submitted the plugin, btw :).

I planned to have these two files packaged in with the plugin. I just forgot to add them to the release. I should be able to fix this later this evening.

**Comment by nicholas-wilcox** [comment]
This has been resolved by #3. I've also added an archive of all the release files to the release itself, as well as some installation instructions to aid users until I am ready to submit this plugin to Obsidian's review process.
